### ISSUE TITLE
[Feature-Request] 720p

### ISSUE THREAD

**Issue by Gunnar-GO** [marty28/DiabloX#3]
Is it possible to use the 720p-Resolution of the XBox ? The game would benefit, since the resolution would be better scaled by modern displays.

**Comment by AJenbo** [comment]
No, the resolution is hard coded. As you have noted DiabloX is based on DevilutionX 1.0.0 and changing resolution didn't become available before 1.1.0.

**Comment by Gunnar-GO** [comment]
As you indicated in the main project, it may find once its way to build on the actual code base on a regular basis.

**Comment by Firestorm-Apocalypse** [comment]
I will start updating the code to 1.5.1, and then submit it here for analysis. Any help is appreciated.

**Comment by AJenbo** [comment]
1.5.1 actually has Xbox support. So this might be as simple as enabling and testing 720p as a resolution :)
